src/data/seq/seq.lean

```diff
@@ -1,4 +1,10 @@
-import data.stream data.lazy_list data.seq.computation logic.basic tactic.interactive
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
```

Comment:
Thanks for the credit, but I don't work for Microsoft. The common practice is to put the first author in the copyright header; the microsoft header only applies to stuff written by Leo and others working at MR.

Reply:
Fixed.